### PR TITLE
Get All By Filter

### PR DESCRIPTION
**Pull request by jitendrajaria**
get all by filter  include filter {inquiryStatus, caseIndex,cityId}

### CODE REVIEW

File: src/main/java/creativei/dao/InquiryCustomDao.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+import creativei.entity.City;
+import creativei.entity.Inquiry;
+import creativei.enums.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+
+public interface InquiryCustomDao extends JpaRepository<Inquiry,Long>{
```
Comment (aayushKhandpur) [inline src/main/java/creativei/dao/InquiryCustomDao.java]: No need to extend repository dao here. Extend following-
Inquirydao extends repositoryDao and the n add inquiryDao autowire into inquiryCustomdaoimpl. Follow the pr I shared as example

**Review by aayushKhandpur — CHANGES_REQUESTED** [review]
Address review

File: src/main/java/creativei/dao/InquiryCustomDao.java (hunk 1 above)
Comment (jitendrajaria) [reply src/main/java/creativei/dao/InquiryCustomDao.java]: issue resolved in commit 75991e6

File: src/main/java/creativei/dao/InquiryCustomDao.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import creativei.enums.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+
+public interface InquiryCustomDao {
+    List<Inquiry> findByInquiryStatus(InquiryStatus status);
+
+    List<Inquiry> getFilteredInquiries(InquiryStatus status, CaseIndex caseIndex, Long cityId);
```
Comment (aayushKhandpur) [inline src/main/java/creativei/dao/InquiryCustomDao.java]: Try using consistent method names. If other naming is findBy.. this should also be named similarly

**Review by aayushKhandpur — CHANGES_REQUESTED** [review]
Address review

File: src/main/java/creativei/dao/InquiryCustomDao.java (hunk 2 above)
Comment (jitendrajaria) [reply src/main/java/creativei/dao/InquiryCustomDao.java]: issue resolved in commit f372b98

**Review by aayushKhandpur — APPROVED** [review]

**Pull request merged**